### DISCUSSION TITLE
pin not properly covered

### DISCUSSION THREAD

**Discussion by laurentc2** [The-OpenROAD-Project/OpenROAD-flow-scripts#457] [Q&A] [answered]
I run a script where all my pins are placed using place_pin. But When I try the global placement, I got the error : 
pin ADD[4] not properly covered 
What does this error mean ? Which parameters should I change in the process config or pin placement ?

Thank you, BRgds,
Laurent

**Comment by maliberty** [comment]
A test case would be helpful.

**Reply by laurentc2** [reply, marked as answer]
I actually solved it : the pins I placed were completely outside the die area.
To have the pins at the border of the die area, they have to be placed at the edge of the die area minus half of the pin size.

BRgds,
Laurent

**Comment by maliberty** [comment]
Objects should be fully inside the border not partially sticking out.
